sources/packages/backend/libs/sims-db/src/entities/notification-message.model.ts

```diff
@@ -30,4 +30,14 @@ export class NotificationMessage extends RecordDataModel {
     nullable: false,
   })
   templateId: string;
+  /**
+   * Address to send the email notification.
```

Comment:
If the comment suggested for the SQL is accepted please ensure that this one will also be updated.